cmd/snap-confine/seccomp-support.c

```diff
@@ -172,19 +189,19 @@ int sc_apply_seccomp_bpf(const char *filter_profile)
 		.len = num_read / sizeof(struct sock_filter),
 		.filter = (struct sock_filter *)bpf,
 	};
-	if (prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &prog)) {
-		perror
-		    ("prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, ...) failed");
-		die("aborting");
+	if (prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, &prog) < 0) {
```

Comment:
I prefer the specificity of `!= 0` rather than `< 0` here and above, but these changes are not wrong.